apps/client/src/theme/variables/ionic.scss

```diff
@@ -7,6 +7,17 @@ http://ionicframework.com/docs/theming/ */
   --ion-background-color: #ffffff;
   --ion-background-color-rgb: 255, 255, 255;
 
+  // Difficulty Colors
```

Comment:
kama dean kommentar einheitlich schriba bitte wia alle anderen in dera file

`/** difficulty **/`

und bitte ganz nach unten schieben zu den rank farben